modules/apps/list-type/list-type-api/src/main/java/com/liferay/list/type/service/ListTypeEntryLocalService.java

```diff
@@ -334,7 +334,8 @@ public PersistedModel getPersistedModel(Serializable primaryKeyObj)
 
 	@Indexable(type = IndexableType.REINDEX)
 	public ListTypeEntry updateListTypeEntry(
-			long listTypeEntryId, Map<Locale, String> nameMap)
```

Comment:
Please see previous comment. It would be nice to have just one buildService commit on this pull request.